src/dbus/server/dbus_object.cpp

```diff
@@ -144,19 +145,19 @@ void DBusObject::GetPropertyMethodHandler(DBusRequest &aRequest)
         auto propertyIter = mGetPropertyHandlers.find(interfaceName);
 
         otbrLogInfo("GetProperty %s.%s", interfaceName.c_str(), propertyName.c_str());
-        VerifyOrExit(propertyIter != mGetPropertyHandlers.end(), error = OT_ERROR_NOT_FOUND);
+        VerifyOrExit(propertyIter != mGetPropertyHandlers.end(), error = OT_ERROR_INVALID_ARGS);
```

Comment:
The client wants to get a property, but the property is not found. I think the original not found is more natural. The purpose of this PR is to reduce logs, how about focusing on this first?

Reply:
fine to me to keep using `OT_ERROR_NOT_FOUND`